L/libint/build_tarballs.jl

```diff
@@ -0,0 +1,60 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libint"
+version = v"2.7.0-beta.5"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/evaleev/libint/releases/download/v2.7.0-beta.5/libint-2.7.0-beta.5.tgz", "38c630d9b4433340079ebc7a93f05d7a285833883327bfc35dd208249e06f05f"),
+    GitSource("https://gitlab.com/libeigen/eigen.git", "21ae2afd4edaa1b69782c67a54182d34efe43f9c"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+cd eigen/
+mkdir build
+cd build/
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} ..
+apk add boost-dev
```

Comment:
Note that we have both [`boost_jll`](https://github.com/JuliaBinaryWrappers/boost_jll.jl) and [`Eigen_jll`](https://github.com/JuliaBinaryWrappers/Eigen_jll.jl).  You should use them as dependencies below, rather than installing them in this way

Reply:
Is there any way to fix this without restarting the wizard process? And would it be visible to cmake in the same way?

Reply:
You don't have to use the wizard to run a build, I actually rarely use it: https://juliapackaging.github.io/BinaryBuilder.jl/dev/FAQ/#I-love-the-wizard,-but-now-I-want-to-break-free:-can-I-build-the-tarballs-without-it?-1

Reply:
Oh, very cool. Thanks!